funnel/templates/schedule_edit.html.jinja2

```diff
@@ -97,10 +100,13 @@
     <script type="text/javascript" src="{{ ASSET_URL }}"></script>
   {%- endassets -%}
   {% assets "js_schedules" -%}
+    <script type="text/javascript" src="https://momentjs.com/downloads/moment.js"></script>
+    <script type="text/javascript" src="https://momentjs.com/downloads/moment-timezone-with-data-10-year-range.js"></script>
     <script type="text/javascript" src="{{ ASSET_URL }}"></script>
     <script type="text/javascript">
       var UPDATE_URL = "{{ project.url_for('update_schedule') }}";
-      var TIMEZONE = {{ timezone }} * 1000;
+      var TIMEZONEOFFSET = {{ timezone_offset }} * 1000;
```

Comment:
If the 1000 is because JS works in microseconds instead of seconds, a comment will help.